ArchiSteamFarm/NLog/ArchiLogger.cs

```diff
@@ -151,6 +152,20 @@ public sealed class ArchiLogger {
 			Logger.Log(logEventInfo);
 		}
 
+		internal void LogInvite(ulong steamID, EAccountType accountType,  [CallerMemberName] string? previousMethodName = null) {
+			if ((accountType == EAccountType.Invalid) || (steamID == 0)) {
+				throw new ArgumentNullException(nameof(accountType) + " || " + nameof(steamID));
+			}
+
+			string loggedMessage = previousMethodName + "() " + Enum.GetName(typeof(EAccountType), accountType) + " " + Strings.Invite + " " + steamID.ToString();
```

Comment:
Same here. In addition to that, I completely miss the point of `Strings.Invite`, just log the content alone without additional localization, this method is already named `LogInvite()` which people will filter against.

Reply:
Moreover, localization here is actually super bad as it can garble the input with stuff people might not want, it's best to keep it as generic as possible. Sure, that's what event properties are for, but there is no need to overcomplicate this.